packages/nodes-base/nodes/Clerk/ClerkTrigger.node.ts

```diff
@@ -98,12 +98,11 @@ export class ClerkTrigger implements INodeType {
 		const { rawBody, headers, body } = this.getRequestObject() as ClerkRequest;
 		const events = this.getNodeParameter('events', []) as ClerkEvent[];
 		const { webhookSecret } = this.getCredentials('clerkApi') as ClerkCredentials;
-		const webhook = new Webhook(webhookSecret);
 
 		let payload;
 
 		try {
-			payload = webhook.verify(rawBody.toString(), headers) as ClerkWebhookPayload;
+			payload = verify.call(this, rawBody.toString(), headers, webhookSecret) as ClerkWebhookPayload;
```

Comment:
@ivov 
Thanks for the work on this, again couple of questions:
1. Would it be a convention in the `GenericFunction` implementations to implicity access `this` and then calling it with `Function.prototype.call` as you do here ? _In my view, to avoid context specific issues, I would pass the node as a dependency._
2. Are we confident adding `verify` as a local implementation ? In the case that `svix` changes the verification algorithm (_which is not uncommon_), the plugin will break for all clients without the capacity to see what is wrong. They will just see a **Unverified webhook call!** error. What is the driving factor of not introducing this dependency which is critical to the functionality of the node ?

Reply:
1. More than a convention, this is a necessity because of how nodes were originally designed. `this` is not the class, but the arg to the method, a bundle of functions of type `IWebhookFunctions`.
2. In the tradeoff between the risk of a local implementation becoming outdated and the implications of adding a lib for every new node, the team prefers a local implementation where possible.